io/zenoh-transport/src/unicast/manager.rs

```diff
@@ -406,12 +422,53 @@ impl TransportManager {
                     #[cfg(feature = "shared-memory")]
                     is_shm: config.is_shm,
                 };
-                let a_t = Arc::new(TransportUnicastInner::make(self.clone(), stc)?);
+
+                // select and create transport implementation depending on the cfg
+                let a_t: Result<Arc<dyn TransportUnicastInnerTrait>, (Error, Option<u8>)> = {
+                    #[cfg(feature = "shared-memory")]
+                    {
+                        match stc.is_shm {
+                            false => {
+                                log::info!("Will use NET transport!");
```

Comment:
I don't think there is the need to log here. Information about the transport being used can is displayed at debug level upon successful establishment.

Moreover `log::info` is highly discouraged in the transport layer.